synapse_data_warehouse/synapse_raw/tables/V2.23.0__add_version_history_to_nodesnapshot.sql

```diff
@@ -0,0 +1,9 @@
+-- Configure environment
+USE ROLE sysadmin;
+USE SCHEMA {{database_name}}.synapse_raw; --noqa: JJ01,PRS,TMP,CP02
+USE WAREHOUSE compute_xsmall;
+
+-- Add `version_history` column
+ALTER TABLE nodesnapshots
```

Comment:
Should we just use a R script and do `CREATE OR ALTER` and take the time to document all the columns

Reply:
And as a general rule of thumb, changes to tables ought to be versioned. Although I'll admit that this reasoning gets a bit convoluted when you consider that we always have the git version (commit sha), so versioning changes with schemachange can seem redundant at times.